src/Microsoft.AspNetCore.Session/LoggingExtensions.cs

```diff
@@ -45,6 +50,26 @@ static LoggingExtensions()
                 eventId: 7,
                 logLevel: LogLevel.Warning,
                 formatString: "Error unprotecting the session cookie.");
+            _sessionLoadingTimeout = LoggerMessage.Define(
+                eventId: 8,
+                logLevel: LogLevel.Warning,
+                formatString: "Loading the session timed out.");
+            _sessionCommitTimeout = LoggerMessage.Define(
+                eventId: 10,
```

Comment:
Why the gap in the event ids? Maybe 7 8 9?

Reply:
Good point, I had another event that was removed.

Reply:
Or maybe we are keeping in line with the recent trend of omitting 9? Windows 8 -> 10, iPhone 8 -> X.